src/components/accounts/instances/InstancesApp.js

```diff
@@ -36,7 +36,7 @@ const Index = ({ children }) => {
     location?.pathname?.split('/').length > 2
       ? location?.pathname?.split('/')[3]
       : '';
-  const [tabValue, setTabValue] = useState(currentPage);
+  const [, setTabValue] = useState(currentPage);
```

Comment:
#1 I'll fix that issue in modal 
![image](https://github.com/zesty-io/website/assets/44116036/a055caa9-a582-4f1a-bced-9f3de6623882)

#2 issue: in the accounts tab in Mobile. I removed the tabValue since its not being used since currentPage is enough to tell where the current tab is selected
![image](https://github.com/zesty-io/website/assets/44116036/0c93cc8e-a79b-4638-af7b-102ff400a804)

Reply:
@darwin808 this will fix the issue in the number 1 
https://github.com/zesty-io/website/pull/2249/commits/dbdd62c4c29b0d2144070f5035d6903c6119f1a9

Reply:
![image](https://github.com/zesty-io/website/assets/44116036/484cd04d-e9dc-4034-8c9d-2884a64a93d8)